mysql-test/suite/innodb/t/bug-76142.test

```diff
@@ -0,0 +1,41 @@
+--source include/have_innodb.inc
+
+--echo #
+--echo # Bug 76142: InnoDB tablespace import fails when importing table w/ different data directory
+--echo #
+
+--let $MYSQLD_DATADIR = `SELECT @@datadir`
+
```

Comment:
Add
--let $CURRENT_DB = `SELECT DATABASE()`
and then use ".../$CURRENT_DB/..." in --copy-file / --move_file paths instead of ".../test/..."